tests/test.js

```diff
@@ -0,0 +1,39 @@
+const assert = require('assert')
+const spawn = require('child_process').spawn
+const uuidV4 = require("uuid/v4")
+const fs = require('fs')
+
+describe('WebUi Generator', () => {
+  let uniqueId = uuidV4()
+  let email = 'fossasia@gmail.com'
+  let args = [
+    "-g", "https://github.com/fossasia/yaydoc.git",
```

Comment:
just to be sure, does running this test actually executes the `generate` script?